Configure.ps1

```diff
@@ -153,31 +161,19 @@ so that they are consistent with the Applications parameters
                             -tenantId $tenantId `
                             -audience $serviceAppIdURI
 
-    # prepare the clean-up
```

Comment:
the Cleanup script can now be ran at any time. We don't return any longer the ids of the created applications. They are found in the AAD from the parameters.